rotkehlchen/chain/ethereum/modules/uniswap/v2/decoder.py

```diff
@@ -157,5 +195,10 @@ def enricher_rules(self) -> list[Callable]:
             self._maybe_enrich_lp_tokens_transfers,
         ]
 
+    def post_decoding_rules(self) -> list[tuple[int, Callable]]:
+        return [
+            (0, self._fix_order_of_events_for_swaps),
```

Comment:
As has been commented in other PR this will execute for all the transactions and is costly. We will refactor the post decoding rule to gain context of the transaction and not execute them always